sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/NetworkStateChangeStorageTest.java

```diff
@@ -88,12 +88,18 @@ public void pendingCreateOperationSuccess() throws JSONException {
         verifyNoMoreInteractions(mHttpClient);
     }
 
+    @Test
+    public void noPendingOperationsWhenSignedOut() {
+        when(mAuthTokenContext.getAccountId()).thenReturn(null);
```

Comment:
This test verifies nothing

Reply:
Why was it marked as resolved?

Reply:
Because there's another test `LocalDocumentStorageTest.getPendingOperationsOnNonUserTable` that verifies and covers what the one you pointed out to was supposed to, but didn't,

Reply:
ok